.github/workflows/pre_release.yml

```diff
@@ -0,0 +1,49 @@
+name: pre-release
+
+on:
+  create
+
+jobs:
+
+  # Make a PR to master from a new branch with changes, and delete the created one
+  update-versions:
+
+    # Only run when a release-v* branch is created, and not by drashbot
+    if: contains(github.ref, 'release-v') && !contains(github.event.sender.login, 'drashbot')
+
+    strategy:
+      matrix:
+        deno: ["1.5.1"]
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Checkout master
+        uses: actions/checkout@v2
+        with:
+          ref: master
+
+      - name: Delete Remote Release Branch
+        run: git push --delete origin ${{ github.ref }}
+
+      - name: Install Deno
+        uses: denolib/setup-deno@v2
+        with:
+          deno-version: ${{ matrix.deno }}
+
+      - name: Bump Versions
+        run: deno run -A ./console/bumper_ci_service.ts --version=${{ github.ref }} # version passed so we know what the new release version will be
+
+      - name: Send Event to Website to Update Versions
+        run: |
+          curl -XPOST -u "${{ secrets.CI_USER_NAME }}:${{ secrets.CI_USER_PAT }}" -H "Accept: application/vnd.github.everest-preview+json" -H "Content-Type: application/json" https://api.github.com/repos/drashland/website/dispatches --data '{"event_type": "update_docs", "client_payload": { "module_to_update": "rhum", "version": "${{ github.ref }}" }}'
```

Comment:
can we break this into lines por favor 😃

Reply:
nvm. idk if breaking it apart into separate lines would work the first time and i want this to work the first time. resolving.